package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "eslint-plugin-sonarjs",
-  "version": "0.25.1",
+  "version": "0.26.0",
```

Comment:
I didn't land this version pull after the last release (https://github.com/SonarSource/eslint-plugin-sonarjs/pull/455)